inputs/LiMetal_PorousSep_oxygen.yaml

```diff
@@ -121,43 +113,86 @@ phases:
   state:
     T: 300.0
     P: 1 atm
-- name: cathode
-  thermo: ideal-condensed
-  elements: [Li, C, O, Mn]
-  species: ['C[cathode]', 'Li2O2[cathode]', 'Cat[cathode]']
-  state:
-    X: {'C[cathode]': 0.80, 'Li2O2[cathode]': 0.00, 'Cat[cathode]': 0.20 }
-  standard-concentration-basis: unity
-  tabulated-species: Li2O2[cathode]
-- name: electrolyte
-  thermo: ideal-condensed
-  elements: [Li, P, F, C, H, O, E]
-  species: ['C3H4O3[elyt]', 'Li+[elyt]', 'PF6-[elyt]', O2(e), LiO2]
-  state:
-    X: {'C3H4O3[elyt]': 0.85464, 'Li+[elyt]': 0.07268,
-      'PF6-[elyt]': 0.07268}
-  standard-concentration-basis: unity
-  reactions: [electrolyte-reactions]
+
 - name: electron
   thermo: electron-cloud
   elements: [E]
   species: [electron]
   state:
     X: {electron: 1.0}
   density: 1.0 kg/m^3
+
 - name: lithium_electrolyte
   thermo: ideal-surface
   species: [(dummy)]
   kinetics: surface
   reactions: [lithium_electrolyte-reactions]
   site-density: 0.01 mol/cm^2
+  
+- name: electrolyte
+  thermo: ideal-condensed
+  elements: [Li, P, F, C, H, O, E]
+  species: ['C10H22O5[elyt]', 'Li+[elyt]', 'TFSI-[elyt]', O2(e), LiO2]
+  state:
+    X: {'C10H22O5[elyt]': 0.85464, 'TFSI-[elyt]': 0.07268,
+      'PF6-[elyt]': 0.07268, 'O2(e)': 0.0, 'LiO2': 0.0} 
+  standard-concentration-basis: unity
+  reactions: [electrolyte-reactions]
+
 - name: air-elyte-interface
   thermo: ideal-surface
   species: [(dummy)]
   kinetics: surface
   reactions: [air_electrolyte-reactions]
   site-density: 0.01 mol/cm^2
 
+- name: oxygen
+  thermo: ideal-gas
+  elements: [O]
+  species: [O2(g)]
+  transport: mixture-averaged
+  state: {T: 300.0, P: 1 atm}
+
+- name: conductor
+  thermo: electron-cloud
+  elements: [C]
+  species: ['C[cathode]']
```

Comment:
This is perhaps counter-intuitive, but the only "species" that you care about, for this phase, i the electron.  Since the carbon does not participate in any reactions, you do not need to model is, explicitly.  The carbon phase, however, does provide or accept electrons for charge-transfer reactions.